### PR TITLE
bnfc 2.9.4

### PR DESCRIPTION
**Pull request by pthariensflame**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by pthariensflame** [comment]
This should be ready for review!

File: Formula/bnfc.rb [hunk 1]
```diff
@@ -45,6 +45,11 @@ def install
 
   test do
     ENV.prepend_create_path "PATH", testpath/"tools-bin"
+    system "cabal", "v2-update"
+    system "cabal", "v2-install",
+           "--jobs=#{ENV.make_jobs}", "--max-backjumps=100000",
+           "--install-method=copy", "--installdir=#{testpath/"tools-bin"}",
+           "alex", "happy"
```
Comment (SMillerDev) [inline Formula/bnfc.rb:52]: If this is needed for the test, shouldn't it be in the install?

File: Formula/bnfc.rb [hunk 2]
```diff
@@ -20,12 +20,12 @@ class Bnfc < Formula
   depends_on "cabal-install" => [:build, :test]
   depends_on "ghc" => [:build, :test]
   depends_on "sphinx-doc" => :build
+  depends_on "agda" => :test
   depends_on "antlr" => :test
+  depends_on "bison" => :test
+  depends_on "flex" => :test
```
Comment (SMillerDev) [inline Formula/bnfc.rb:26]: Why not the system ones?

Comment (pthariensflame) [reply Formula/bnfc.rb:26]: The system ones error on the test now; they used to be fine but now Apple’s bison gives a syntax error on the intermediate `.y` file.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**